src/karmen_backend/openapi/swagger.yaml

```diff
@@ -0,0 +1,1544 @@
+        - $ref: '#/components/parameters/organization_uuid_param'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                target:
+                  type: string
+                  enum: [ off]
+            examples:
+              disable:
+                value:
+                  target: off
+      responses:
+        '204':
+          description: OK
+        '400':
+          description: >-
+            Target is not off
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/api_err_response'
+              example:
+                message: Invalid target
+
+        '500':
+          description: >-
+            Printer refused to control motors
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/api_err_response'
+              example:
+                message: Cannot control motors
+
+  '/organizations/{org_uuid}/printers/{printer_uuid}/extrusion':
+    post:
+      summary: Extrude or retrract filament
+      operationId: server.routes.printers.control_extrusion
+      tags:
+        - printers
+      parameters:
+        - $ref: '#/components/parameters/printer_uuid_param'
+        - $ref: '#/components/parameters/organization_uuid_param'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                target:
+                  type: string
+                  enum: [on, off]
+            examples:
+              turn on:
+                value:
+                  target: on
+              turn off:
+                value:
+                  target: off
+      responses:
+        '204':
+          description: OK
+        '400':
+          description: >-
+            Target is not on or off
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/api_err_response'
+              example:
+                message: Invalid target
+
+        '500':
+          description: >-
+            Printer refused to control fan
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/api_err_response'
+              example:
+                message: Cannot control fan
+
+  '/organizations/{org_uuid}/printers/{printer_uuid}/temperatures/{part_name}':
+    post:
+      summary: Set temperatures for part
+      operationId: server.routes.printers.control_tool_temperature
+      tags:
+        - printers
+      parameters:
+        - $ref: '#/components/parameters/printer_uuid_param'
+        - $ref: '#/components/parameters/organization_uuid_param'
+        - in: path
+          name: part_name
+          schema:
+            type: string
+            enum: [tool0, tool1, bed]
+          required: true
+          description: Part to set semperature
```

Comment:
Component to set temperature for

Reply:
I would stay with "part", like octoprint does.